velox/type/Variant.cpp

```diff
@@ -257,19 +259,22 @@ std::string variant::toJson(const TypePtr& type) const {
       folly::json::escapeString(str, target, getOpts());
       return target;
     }
-    case TypeKind::HUGEINT:
-      VELOX_CHECK(type && type->isLongDecimal());
```

Comment:
This should remain a `VELOX_CHECK` since int128_t native type is not officially supported.

Reply:
Resolved.